CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Added
 
+- Boilerplate for the Device Claiming Server component.
```

Comment:
Is this relevant for stack operators ? This is a message of interest to stack developers, not stack operators.